pxtcompiler/emitter/hexfile.ts

```diff
@@ -1088,13 +1088,16 @@ _stored_program: .hex ${res}
     }
 
     function assembleAndPatch(src: string, bin: Binary, opts: CompileOptions, cres: CompileResult) {
-        if (opts.extinfo.disabledDeps) {
+        const dummy = opts.extinfo.disabledDeps
```

Comment:
extinfo could be undefined if the C++ build failed?

Reply:
I don't believe we get here when the build failed; there was an unguarded reference down below for a while